frontend/src/core/user/components/UserMenu.css

```diff
@@ -12,7 +12,7 @@
 .user-menu .selector .menu-icon,
 .user-menu .menu li.details img {
     border: 2px solid #4d5967;
-    border-radius: 100%;
+    border-radius: 4px;
```

Comment:
Nice! You still need to make this change for the avatars in the [Filter menu](https://mozilla-l10n.github.io/localizer-documentation/tools/pontoon/search_filters.html).

Reply:
Sorry, missed this one :)

Reply:
BTW, the filters button is not obvious, it was hard to find it for me.
![image](https://user-images.githubusercontent.com/52250201/66926886-cd7f9b80-effc-11e9-963a-0953c703de9c.png)
Should it be an image like in the documentation screenshot?